### PR TITLE
For #19270 - Added a string resource and removed Hardcoded string.

### PR DESCRIPTION
**Pull request by priyanka0906**
fixes : #19270 
Added a string resource for the hardcoded string "close".
### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

**Comment by priyanka0906** [comment]
I have added string resource only in one string.xml file. Please confirm if it should be added in other string.xml files. Thank you.

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1713,4 +1713,8 @@
 
     <!-- Content description radio buttons with a link to more information -->
     <string name="radio_preference_info_content_description">Click for more details</string>
+
+    <!-- Content description for privacy content close button -->
+    <string name="privacy_content_close_button_content_description">close</string>
```
Comment (eliserichards) [inline app/src/main/res/values/strings.xml:1718]: Nit: even though the old string is "close", let's capitalize this "Close" for consistency 😄

**Review by eliserichards — CHANGES_REQUESTED** [review]

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (priyanka0906) [reply app/src/main/res/values/strings.xml:1718]: okay.

Comment (priyanka0906) [reply app/src/main/res/values/strings.xml:1718]: @eliserichards also if it should be added to other strings.xml files

Comment (eliserichards) [reply app/src/main/res/values/strings.xml:1718]: Adding it here in the main file is all you need. Our translators will add it to the other files as localized strings become available :)

Comment (priyanka0906) [reply app/src/main/res/values/strings.xml:1718]: > Adding it here in the main file is all you need. Our translators will add it to the other files as localized strings become available :)

ohhkk, Thanks for confirming☺

Comment (priyanka0906) [reply app/src/main/res/values/strings.xml]: @eliserichards I have capitalizes Close.

**Review by eliserichards — APPROVED** [review]
Great job! 🚀

**Comment by priyanka0906** [comment]
> Great job! 🚀

@eliserichards Thank you😊

**Comment by eliserichards** [comment]
Closing in favor of https://github.com/mozilla-mobile/fenix/pull/19500

**Pull request closed without merging**